src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx

```diff
@@ -19,7 +19,9 @@ As a New Relic user, you may have more than one user record using the same email
 
 ## Switch between user records [#switch]
 
-When you log in to New Relic, you can go to the user menu to switch to other user records that have the same email address. 
+When you log in to New Relic, you can go to the [user menu](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) to switch to other user records that have the same email address. When you choose a user record from the menu, you'll be prompted to log in for that user record (either directly to New Relic or via your identity provider).
+
+ If the desired user record has a manual login (not single sign-on), you'll be  records for non-SSO users.
```

Comment:
There appears to be an extra space and/or a type before "records for non-SSO users."